### PR TITLE
ISPN-16795 NullPointerException when gauges is set to false

### PR DESCRIPTION
**Pull request by pruivo**
https://issues.redhat.com/browse/ISPN-16795

### CODE REVIEW

File: server/tests/src/test/java/org/infinispan/server/functional/rest/RestMetricsWithoutGaugeResourceIT.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+import org.infinispan.server.test.core.ServerRunMode;
+import org.infinispan.server.test.junit5.InfinispanServerExtension;
+import org.infinispan.server.test.junit5.InfinispanServerExtensionBuilder;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+public class RestMetricsWithoutGaugeResourceIT {
+
+   private static final int NUM_SERVERS = 3;
```
Comment (ryanemerson) [inline server/tests/src/test/java/org/infinispan/server/functional/rest/RestMetricsWithoutGaugeResourceIT.java:15]: Why do we need 3 servers?

Comment (tristantarrant) [reply server/tests/src/test/java/org/infinispan/server/functional/rest/RestMetricsWithoutGaugeResourceIT.java:15]: Agreed: if you need clustered, 2 is enough.

**Review by jabolina — APPROVED** [review]
Seems good. I've replayed CI just to be on the safe side.

**Comment by jabolina** [comment]
@pruivo Seems it needs a rebase

**Pull request merged**